.eslintrc.json

```diff
@@ -1,11 +1,16 @@
 {
-  "extends": "google",
+  "extends": [
+    "google",
+    "plugin:eslint-plugin/all"
```

Comment:
What's the reason for using `all` instead of `recommended`?

Reply:
Personally I consider ALL of the [eslint-plugin-eslint-plugin](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin#rules) rules suitable for general usage, even though a handful of them aren't `recommended` because they could be considered somewhat stylistic/opinionated. And as you can see in the diff of the PR, the changes are fairly minimal / agreeable.

But if anyone objects, it's no problem to switch to `recommended` either now or later.